pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -24,6 +24,7 @@ type MTOMilestone {
   Note, this data will be cleared if other is removed as an option
   """
   facilitatedByOther: String
+  assignedTo: UserAccount
```

Comment:
I think we should split this, we typically allow access to the raw data. Since assignedTo is the name of the DB field, I'd suggest we have `assignedTo` as well as `assignedToUserAccount`. So if someone just wants the uuid, they can get it without having to invoke the loader